src/adapter/src/catalog.rs

```diff
@@ -6603,6 +6615,14 @@ impl Catalog {
                     }
                     ObjectId::Role(_) => unreachable!("roles have no owner"),
                 },
+                Op::UpdateClusterConfig { id, name, config } => {
+                    builtin_table_updates.push(state.pack_cluster_update(&name, -1));
+                    let cluster = state.get_cluster_mut(id);
+                    cluster.config = config;
+                    tx.update_cluster(id, cluster)?;
+                    builtin_table_updates.push(state.pack_cluster_update(&name, 1));
+                    info!("update cluster {}", name);
+                }
```

Comment:
There doesn't seem to be a specific rule (that I'm aware of) about what goes in the audit log. This seems like it would be useful to have in the audit log though, so I would recommend adding it.

Reply:
Filed #19978 to implement this.